test/Infrastructure/LeanCode.Firebase.FCM.Tests/MsSqlTokenStore/MsSqlPushNotificationTokenStoreTests.cs

```diff
@@ -12,7 +12,7 @@ public class MSSQLPushNotificationTokenStoreTests : IAsyncLifetime
     public async Task Gets_freshly_saved_token()
     {
         const string Token = "token";
-        var uid = "UserId";
+        const string uid = "UserId";
```

Comment:
And now nit on naming - consts use PascalCase instead of normal variable camelCase